include/vara/Feature/FeatureModelTransaction.h

```diff
@@ -241,8 +310,22 @@ class FeatureModelCopyTransactionBase {
         std::move(NewFeature))(*FM);
   }
 
+  Feature *addRelationshipImpl(Relationship::RelationshipKind Kind,
+                               Feature *GroupRoot) {
+    if (!FM || !GroupRoot) {
+      return nullptr;
+    }
+
+    return FeatureModelModification::make_modification<
+        AddRelationshipToFeature>(Kind, TranslateFeature(GroupRoot))(*FM);
+  }
+
 private:
   std::unique_ptr<FeatureModel> FM;
+
+  Feature *TranslateFeature(const Feature *F) const {
```

Comment:
```suggestion
  Feature *MapToCopiedFM(const Feature *F) const {
```
It think we should call the method something with `map`, about the Copied part I'm not so sure^^